### PR TITLE
Create jai

### PR DESCRIPTION
**Pull request by iamthannos**

### CODE REVIEW

File: jai [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Hii
```
Comment (ns-jshaik) [inline jai:1]: Hii

File: jai [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+THis is jai
```
Comment (ns-jshaik) [inline jai:2]: Add more data

Comment (ns-jshaik) [reply jai:2]: Cleanup the code

File: jai (hunk 1 above)
Comment (iamthannos) [reply jai:1]: Hello

File: jai (hunk 2 above)
Comment (iamthannos) [reply jai:2]: Ok

**Review by ns-jshaik — APPROVED** [review]

**Pull request merged**